src/Bridges/Bridges.jl

```diff
@@ -264,6 +265,9 @@ function runtests(
     final_touch(model)
     # Should be able to call final_touch multiple times.
     final_touch(model)
+    if print_inner_model
+        print(inner)
```

Comment:
That's useful

Reply:
Yeah I always use it when writing tests, but I normally just manually edit the file to put the `print` in.